lib/team_builder.rb

```diff
@@ -44,10 +44,11 @@ def build_single_team(team_name)
   end
 
   def build_all_teams
-    static_config.map do |_, team_config|
+    static_config.map do |key, team_config|
       if team_config["repos"].nil?
         team_config["repos"] = govuk_team_repos(team_config["channel"])
       end
+      team_config["ci_checks"] = is_govuk_team?(key)
```

Comment:
Another refactor commit idea - can we get Seal to raise an exception if it encounters a GOV.UK team where the repos have been added to `config/alphagov.yml`?

Reply:
I've added a commit to implement your suggestion.